src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java

```diff
@@ -102,7 +101,7 @@ public static Options defaultOptions() {
                     GraphQLContext.getDefault(),
                     Locale.getDefault(),
                     Integer.MAX_VALUE,
-                    Integer.MAX_VALUE,
+                    100_000,
```

Comment:
We should have a JVM static way of getting / setting this default - like we do in other places

See `graphql.parser.ParserOptions#MAX_WHITESPACE_TOKENS` for inspiration